src/main/java/hudson/plugins/sshslaves/SSHLauncher.java

```diff
@@ -1458,6 +1469,37 @@ public void setTcpNoDelay(boolean tcpNoDelay) {
         this.tcpNoDelay = tcpNoDelay;
     }
 
+    public String getWorkDir() {
+        return workDir;
+    }
+
+    @DataBoundSetter
+    public void setWorkDir(String workDir) {
+        this.workDir = Util.fixEmptyAndTrim(workDir);
+    }
+
+    /**
+     * @param workingDirectory The Working directory set on the configuration of the node.
+     * @return <p>the remoting parameter to set the workDir,
```

Comment:
I'm pretty sure this return tag will include all the lines without needing the `<p>` wrapper. Otherwise, perhaps it'd be better to put the docs above the tags?